035-HubAndSpoke/Student/Resources/routing-cheat-sheet

```diff
@@ -0,0 +1,73 @@
+# Description
```

Comment:
It doesn't look like this file has an extension.

Since it is a markdown file, it should be named `routing-cheat-sheet.md`

Reply:
Also is this file linked to from any other files? It doesn't seem so.

Reply:
@gfilicetti - fixed both